README.md

```diff
@@ -24,19 +25,28 @@ Groups that this package defines are listed below.
 - privileged
   - For processes that need to be able to access privileged data.
 - sailfish-system
-  - For users and processes that need to be able to control the device.
+  - For users that need to be able to control the device settings.
     Device owner belongs to this group.
 - sailfish-mdm
-  - For processes that need to be able to access MDM APIs.
+  - For sailfish-mdm user and processes that need to be able to access
+    MDM APIs.
 - sailfish-radio
   - For processes that need to be able to access mobile network related APIs.
+- sailfish-calls
+  - For users that need to be able to make calls.
+- sailfish-messages
+  - For users that need to be able to send SMS and MMS.
 - sailfish-alarms
-  - For processes that need to be able to access shared alarm events.
+  - For users that need to be able to access shared alarm events.
 - sailfish-datetime
-  - For processes that need to be able to modify clock settings.
+  - For users that need to be able to modify clock settings.
 - timed
-  - For timed process, required for changing system default timezone
+  - For users and timed process, required to change system default
+    timezone.
+- sailfish-actdead
+  - For sailfish-actdead user.
+- oneshot
```

Comment:
Is this actually used? To me it looks like it's not anymore.

Reply:
Same here. None of the dirs afais has oneshot group: 

> find / -group oneshot

Reply:
Go ahead and nuke this.